pyproject.toml

```diff
@@ -38,6 +38,7 @@ bleak-winrt = { version = "^1.2.0", markers = "platform_system=='Windows'", pyth
 "winrt-Windows.Foundation.Collections" = { version = "^2", markers = "platform_system=='Windows'", python = ">=3.12" }
 "winrt-Windows.Storage.Streams" = { version = "^2", markers = "platform_system=='Windows'", python = ">=3.12" }
 dbus-fast = { version = ">=1.83.0, < 3", markers = "platform_system == 'Linux'" }
+bumble = "^0.0.201"
```

Comment:
I think this should be an optional dependency (i.e. only installed if `pip install bleak[bumble]`). Otherwise it is going to bring in a lot of other dependencies that most people don't need.